CMakeLists.txt

```diff
@@ -48,3 +48,13 @@ add_custom_target(manpage ALL DEPENDS tmatrix.6.gz)
 # Install tmatrix binary and manpage.
 install(TARGETS tmatrix DESTINATION bin)
 install(FILES ${PROJECT_BINARY_DIR}/tmatrix.6.gz DESTINATION share/man/man6)
+
+# Completions
```

Comment:
`Completions` -> `Install bash and zsh completion scripts.`

Also swap the order of the zsh and bash install commands.

Reply:
Sure.